themes/default/content/registry/packages/dynatrace/installation-configuration.md

```diff
@@ -10,7 +10,6 @@ The Pulumi Dynatrace provider is available as a package in all Pulumi languages:
 
 * JavaScript/TypeScript: [`@lbrlabs/pulumi-dynatrace`](https://www.npmjs.com/package/@lbrlabs/pulumi-dynatrace)
 * Python: [`dynatrace`](https://pypi.org/project/lbrlabs-pulumi-dynatrace/)
-* Go: [`github.com/lbrlabs/pulumi-dynatrace/sdk/go/dynatrace`](https://pkg.go.dev/github.com/lbrlabs/pulumi-dynatrace/sdk)
```

Comment:
why isn't this https://github.com/lbrlabs/pulumi-dynatrace/tree/main/sdk/go

Reply:
niice, will update